tests/toolchain/abigen-pass/kv_table_key_type.cpp

```diff
@@ -0,0 +1,57 @@
+#include <eosio/eosio.hpp>
+
+class [[eosio::contract]] kv_table_key_type : public eosio::contract {
+public:
+   using contract::contract;
+
+   struct my_struct {
+      eosio::name primary_key;
+      std::string foo;
+      uint64_t bar;
+
+      std::string fullname;
+      uint32_t age;
+
+      std::pair<int, int> a;
+      std::optional<float> b;
+      std::list<double> c;
+      std::vector<int> d;
+      std::variant<int, bool, float> e;
+
+      eosio::non_unique<std::string, uint32_t> non_unique_name;
```

Comment:
Sorry to ask.  But we are removing the alias `eosio::non_unique` can you change these occurrences in this file to `std::tuple` the rest of your code should work as is (let me know if it doesn't).

Reply:
Updated the test. It works.